### ISSUE TITLE
can editor.getValue();  get all properties?

### ISSUE THREAD

**Issue by Ferossgp** [jdorn/json-editor#347]
Like this:
<pre>
    "age": {
      "type": "integer",
      "default": {{Here Is new Value}},
}
</pre>

**Comment by Z3roCoder** [comment]
GetValue() returns your whole JSON Data as JSON Object.

Lets say you have:

    {
        "properties": {
            "age": {
                "type": "integer",
                "default": 12
            }
        }
    }

as Value. You would do "var jData = editor.getValue()" and wou would be able to access the data via "console.log(jData.properties["age"];"

**Comment by Ferossgp** [comment]
yes, but I need to save this file and edit after some times. And do this often from UI.

**Comment by Z3roCoder** [comment]
You mean the Schema right?

**Comment by Ferossgp** [comment]
yes

**Comment by Z3roCoder** [comment]
Then you would need to get the Schema and not the Value. Look into the documentation, there are methods listed how to get the current schema.

**Comment by Z3roCoder** [comment]
You can easily create a div, set the content of the div to the JSON Editor schema and on buttonclick set the schema from the div as the schema of the editor. Look at the source code of this site (There it is done as an example): http://jeremydorn.com/json-editor/
